### DISCUSSION TITLE
use Carousel component for text

### DISCUSSION THREAD

**Discussion by Marjan-bkh** [facultyai/dash-bootstrap-components#1033] [Q&A]
Hi there, 
Is it possible to use the Carousel component for text? I want to show some text in the slider as top news. How can I do this?

**Comment by Blockheadsuper** [comment]
https://community.plotly.com/t/dash-bootstrap-components-carousel-with-multiple-cards/55467/2

Looks like the answer was no 3 years ago. After searching around the web it seems like it is possible for a bootstrap carousel to contain something other than an img. See here: https://codingyaar.com/bootstrap-4-carousel-multiple-items-responsive/#carouselExampleControls

However, that doesn't change the fact the dbc only supports images in carousels.

**Reply by Blockheadsuper** [reply]
Also, seems like dash-mantine allows for other content in carousels. https://www.dash-mantine-components.com/components/carousel
